apps/itest/lib/account.ex

```diff
@@ -84,7 +84,7 @@ defmodule Itest.Account do
     {:ok, address}
   end
 
-  defp hash(message), do: ExthCrypto.Hash.hash(message, ExthCrypto.Hash.kec())
+  defp hash(message), do: elem(ExKeccak.hash_256(message), 1)
```

Comment:
you selecting the second element of the tuple. but in case of an error, it will be a reason atom. What do you think about explicitly raising an error?

Reply:
from reading the nif, I don't see how this could return an error tuple, it always raises

Reply:
https://github.com/tzumby/ex_keccak/blob/master/test/ex_keccak_test.exs#L24

Reply:
ahhh, missed this:
```
let data: Binary = match args[0].decode() {
        Ok(binary) => binary,
        Err(_error) => return (atoms::error(), atoms::invalid_type()).encode(env),
    };
```